data/user-groups.yml

```diff
@@ -685,6 +685,9 @@
       position:
         lat: 18.5204303
         lng: 73.8567437
+    - name: Punjab Kotlin User Group
```

Comment:
Please add positions :) Soon the build should fail if positions are not present: https://youtrack.jetbrains.com/issue/KT-49574
There's a script that fills in positions automatically: https://github.com/JetBrains/kotlin-web-site/blob/master/scripts/user_groups_geolocator.py but I personally didn't manage to run it.